usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -7140,6 +7234,57 @@ CK_RV ep11tok_check_single_mech_key(STDLL_TokData_t *tokdata, SESSION * session,
     return rc;
 }
 
+struct ECDSA_OTHER_MECH_PARAM {
+    CK_MECHANISM mech;
+    ECSG_Var_t param;
+};
+
+static CK_RV ep11tok_ecdsa_other_mech_adjust(CK_MECHANISM *mech,
+                                      struct ECDSA_OTHER_MECH_PARAM *mech_ep11)
+{
+    CK_IBM_ECDSA_OTHER_PARAMS *param;
+
+    if (mech->mechanism != CKM_IBM_ECDSA_OTHER)
```

Comment:
Here it must be guaranteed that mech is never null

Reply:
Yes this is guaranteed, this is the caller code:
```
    if (mech->mechanism == CKM_IBM_ECDSA_OTHER) {
        rc = ep11tok_ecdsa_other_mech_adjust(mech, &mech_ep11);
```
So if mech is NULL it would crash there already, but its caller already checks that mech is not null.